pcdsdevices/inout.py

```diff
@@ -126,12 +126,16 @@ class InOutRecordPositioner(StateRecordPositioner, InOutPositioner):
     __doc__ += basic_positioner_init
 
 
-class TTReflaser(InOutRecordPositioner):
+class Reflaser(InOutRecordPositioner):
+    """Simple ReferenceLaser with In/Out States"""
+    _icon = 'fa.empire'
+    __doc__ += basic_positioner_init
+
+
+class TTReflaser(Reflaser):
     """
     Motor stack that includes both a timetool and a reflaser.
     """
-    __doc__ += basic_positioner_init
-
```

Comment:
Did you mean to remove these lines?

Reply:
It was moved to the base `Reflaser` class as it pertains to both.

Reply:
But it is immediately overriden by the new subclass